scripts/dataset_processing/ljspeech/create_manifests_and_textfiles.py

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Creates the manifest and .txt transcript files for an LJSpeech split.
-The manifest will be used for training, and the .txt files are for the MFA library to find.
+This script creates NeMo manifests which can be used for training models on LJSpeech (split is taken from https://github.com/NVIDIA/tacotron2).
+It optionally saves transcripts in .txt files (can be used for extracting durations via MFA library).
 """
```

Comment:
Let's call your new download script in this script as well if LJSpeech is not found.

Reply:
Done.